exemplar-crd/service_binding.go

```diff
@@ -93,15 +93,22 @@ type ServiceBindingCondition struct {
 
 // ServiceBindingStatus defines the observed state of ServiceBinding
 type ServiceBindingStatus struct {
-
-	// +kubebuilder:validation:MinItems=1
```

Comment:
I'm pretty sure this now in violation of the spec:

>  At least one condition containing a type of Ready must be defined.

Reply:
the status needs to be empty when the resource is created. The status is updated as the resource is reconciled. Defining the condition needs to be scoped to the reconciler and not the creator.

Reply:
Then the spec should be updated to properly reflect this behavior.  Go ahead and do it as part of this PR.

Reply:
updated